Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,24 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    return unless formula.prefix.directory?
+
+    Dir.glob("#{formula.prefix}/**/*").each do |file|
+      path = Pathname.new file
+      next if path.directory?
+      basename = path.basename
+      return if !Metafiles.copy? basename.to_s
```

Comment:
I've pushed that change here so people can check I'm not missing anything obvious.